#162488889 return author details in trending articles and get all articles route

ucheg6 opened the pull request:
#### What does this PR do?
- return author details in trending articles and get all articles route

#### Description of Task to be completed?
- Include author details when returning articles

#### How should this be manually tested?
- git clone `https://github.com/andela/haven-ah-backend.git`.
- Install `node` if not installed.
- ensure that PostgreSQL is running on your local machine.
- run `npm install` to install the npm packages.
- run `npm test` to run the unit and integration tests.

To test on Postman
- run `npm run start-dev`
- On postman run GET`localhost:5000/api/articles`.
#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#162488889](https://www.pivotaltracker.com/story/show/162488889)

#### Screenshots (if appropriate)

seyi-adeleke commented on server/middlewares/authorization.js:
```diff
@@ -71,17 +71,19 @@ class authorization {
-      return badHttpResponse(response, 401, 'You are not permitted to complete this action');
-    }
+    request.articleId = articleId;
+    // request.isAuthorized = userId === request.userId; // BOOLEAN
```
please remove commented out code

ucheg6 replied:
thanks. That has been done

ucheg6 committed:
bg(return-author): return author details

return author details in trending articles and get all articles route
[finishes #162488889]
